### PR TITLE
chore(etcd): use localpv by default

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Adds a localpv template for etcd with the manual storage class
When configured with the manual storage class use the localpv yaml
Enable localpv persistence by default for develop and release profiles
For tests this can be enabled by adding '-s "etcd.persistence.enabled=true"' to the generate script
(remember to delete the etcd pvc's created by the statefulset)
Trim trailing whitespace from the etcd yaml files

### CODE REVIEW

**Review by paulyoong — APPROVED** [review]

File: chart/templates/mayastor-daemonset.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
         imagePullPolicy: {{ .Values.mayastorImagePullPolicy }}
         env:
         - name: RUST_LOG
-          value: mayastor={{ .Values.mayastorLogLevel }}
+          value: info,mayastor={{ .Values.mayastorLogLevel }}
```
Comment (arne-rusek) [inline chart/templates/mayastor-daemonset.yaml:38]: Thank you!

**Review by arne-rusek — APPROVED** [review]
LGTM

**Comment by gila** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
oh I was making a slight change but it's already merged :D

**Pull request merged**